webdev4.html

```diff
@@ -0,0 +1,165 @@
+    <style>
+        html,
+        body {
+            width: 100%;
+            min-height: 100%;
+        }
+
+        body {
+            padding: 0;
+            margin: 0;
+        }
+
+        #map {
+            top: 60px;
+        }
+
+        #header {
+            top: 0;
+            width: 100%;
+            z-index: 9999999;
+            position: fixed;
+            background-color: cadetblue;
+            color: aliceblue;
+            font-family: monospace;
+            padding-left: 20px;
+
+        }
+
+        #header * {
+            display: inline-block;
+            margin-right: 20px
+        }
+    </style>
+
+</head>
+
+<body>
+    <!--header with map title, class, and my name with link to Leaftlet-->
+    <header id="header">
+        <h1>
+            Voter Precincts in Duluth
+        </h1>
+        <h3>
+            <a href="https://leafletjs.com/">
+                I built this using Leaflet
+            </a>
+        </h3>
+        <h4>
+            Kasey Rankl | GIS 4533 | Fall 2020
+
+        </h4>
+    </header>
+
+    <div id="map" style="width: 100vw; height: 100vw"></div>
+    <!-- Our web map and content will go here -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script>
+    <script>
+        // Create variable to hold map element, give initial settings to map
+        var map = L.map('map', {
+            center: [46.567, -92.1005],
+            zoom: 11
+        });
+        /* Add OpenStreetMap tile layer to map element
+        L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+            attribution: '© OpenStreetMap'
+        }).addTo(map);
+        */
+
+        //add simple basemap
+        var Stamen_TonerLite = L.tileLayer('https://stamen-tiles-{s}.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}{r}.{ext}', {
+            attribution: 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a> &mdash; Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+            subdomains: 'abcd',
+            minZoom: 0,
+            maxZoom: 20,
+            ext: 'png'
+        }).addTo(map);
+
+        // load GeoJSON from an external file        
+        $.getJSON("data/duluth_precincts_WGS84.geojson.txt", function(data) {
+
+            // add GeoJSON layer to the map once the file is loaded
+            L.geoJson(data, {
+                style: function(feature) {
+
+                    // calculate party percent of vote using function
+                    var votePercents = calculatePartyPct(feature);
+
+                    // assign colors from the ColorBrewer red/blue diverging scale
+                    //percentage cutoffs from https://kristofj-umd.github.io/webdev-f19/assignments/webdev4.html
```

Comment:
Good citation.
Even better that it's not a complete replication of the code, and that you made your own tweaks to it.